tests/integration/misc/test_markdown_highlight.py

```diff
@@ -0,0 +1,28 @@
+# -*- coding: utf-8 -*-
+
+import dash
+import dash_core_components as dcc
+import dash_html_components as html
+
+
+md_text = '''```python
+import dash
+print('hello, world!')
+```'''
+
+
+def test_msmh001_no_window_variable(dash_dcc):
+    app = dash.Dash(__name__)
+    app.layout = html.Div(dcc.Markdown(md_text))
+    dash_dcc.start_server(app)
+
+    window_hljs = dash_dcc.driver.execute_script('return window.hljs')
+    assert window_hljs is None
+
+
+def test_msmh001_window_override(dash_dcc):
+    app = dash.Dash(__name__)
+    app.layout = html.Div(dcc.Markdown(md_text))
+    dash_dcc.start_server(app)
+
+    dash_dcc.driver.execute_script('window.hljs = {highlightBlock: (block) => {block.innerHTML="hljs override"}};')
```

Comment:
Here we assign `window.hljs` but nothing in this test demonstrate that it is being used.
Could we add a `dash_dcc.percy_snapshot('<name of snapshot>')` or similar to see that the result is what was forced.